### ISSUE TITLE
FIX BUG !!!

### ISSUE THREAD

**Issue by GamerSoiCon** [davidgamingzz/MiningRewards#5]

**Comment by GamerSoiCon** [comment]
I use worldprotect to protect the world when I block (protected) I get rewards from MiningReward

**Comment by GamerSoiCon** [comment]
sr for bad english

**Comment by davidgamingzz** [comment]
I don’t think it should be able to do that. I made sure that it’s not possible. I also have no clue about how WorldProtect works.
